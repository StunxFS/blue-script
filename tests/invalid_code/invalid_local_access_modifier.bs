fn main() {
    pub var a = 1
    pub const b = 2
}