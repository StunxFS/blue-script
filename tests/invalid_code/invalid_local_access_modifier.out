tests/invalid_code/invalid_local_access_modifier.bs:2:13: error: local variables cannot have access modifier
tests/invalid_code/invalid_local_access_modifier.bs:3:9: error: local constants cannot have access modifier
